conduct/enforcement.md

```diff
@@ -0,0 +1,208 @@
+# Jupyter Code of Conduct - Enforcement Manual
+
+This is the enforcement manual followed by Jupyter's Code of Conduct
+Committee. It's used when we respond to an issue to make sure we're consistent
+and fair.
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee"), currently comprised of [*note:* still gathering volunteers, this will be filled in as they become available and prior to final upload into the rendered site]:
```

Comment:
Another way to do it is to make this generic text, and add the names of committee members in https://github.com/jupyter/governance/blob/master/people.md, just like we do with the Steering Council.